CONTRIBUTING.md

```diff
@@ -48,6 +48,10 @@ To clone and build iTwinUI, you'll need [Git](https://git-scm.com), [Node 18+](h
 
 _Before running this command, make sure Docker is running. See [Visual testing](#visual-testing-css) (CSS and React) sections below for more details._
 
+### To run all tests for a specific component
+
+`pnpm test [component-name]` e.g. `pnpm test Alert`
```

Comment:
Is there any restriction of the case of the component-name? E.g. "alert" vs "Alert"? If so, it would be better to mention it.

Reply:
Yes it needs to be properly capitalized. I'll add a reminder